homeassistant/components/yeelight/__init__.py

```diff
@@ -324,6 +326,22 @@ def _update_properties(self):
 
         return self._available
 
+    def _get_capabilities(self):
+        """Read new properties from the device."""
+        if not self.bulb:
+            return
```

Comment:
Can/should this ever happen?

Reply:
Probalby not, copied from _get_properties will cleanup a bit